src/components/MainPage/Footer/Sections/ContactInfo.jsx

```diff
@@ -1,24 +1,43 @@
-import React from 'react'
-import { InstagramOutlined } from '@ant-design/icons'
-import MessengerIcon from '../social-media/messenger.png'
-import InstagramIcon from '../social-media/insta.png'
-import TelegramIcon from '../social-media/telegram.png'
-import FacebookIcon from '../social-media/facebook.png'
-import EmailIcon from '../social-media/email.svg'
-import PhoneIcon from '../social-media/phone.png'
+import React from "react";
+import { InstagramOutlined } from "@ant-design/icons";
+import MessengerIcon from "../social-media/messenger.png";
+import InstagramIcon from "../social-media/insta.png";
+import TelegramIcon from "../social-media/telegram.png";
+import FacebookIcon from "../social-media/facebook.png";
+import EmailIcon from "../social-media/email.svg";
+import PhoneIcon from "../social-media/phone.png";
 
 export default function ContactInfo() {
   return (
-    <div className='contact-section'>
-        <h3>Contact Information</h3>
-        <nav className='contact-nav'>
-          <a href=''><img src={InstagramIcon} className='social-icon'/></a> 
-          <a href=''><img src={MessengerIcon} className='social-icon'/></a> 
-          <a href=''><img src={TelegramIcon} className='social-icon'/></a>
-          <a href=''><img src={FacebookIcon} className='social-icon'/></a> 
-          <a href=''><img src={EmailIcon} className='social-icon'/></a> 
-          <a href=''><img src={PhoneIcon} className='social-icon'/></a> 
-        </nav>
+    <div className="contact-section">
+      <h3>Contact Information</h3>
+      <nav className="contact-nav">
+        <a href="">
```

Comment:
Lets use constant here.
It's supposte to be an array of object with structure:
`
[
{
icon: string,
className: string, 
href: string,
target: string
}
]
`
We should map this object here instead

Reply:
Something like this?
import React from "react";
import MessengerIcon from "../social-media/messenger.png";
import InstagramIcon from "../social-media/insta.png";
import TelegramIcon from "../social-media/telegram.png";
import FacebookIcon from "../social-media/facebook.png";
import EmailIcon from "../social-media/email.svg";
import PhoneIcon from "../social-media/phone.png";

const contactInfo = [
  {
    icon: InstagramIcon,
    className: "social-icon",
    href: "https://www.instagram.com",
    target: "_blank",
  },
  {
    icon: MessengerIcon,
    className: "social-icon",
    href: "https://www.messenger.com",
    target: "_blank",
  },
  {
    icon: TelegramIcon,
    className: "social-icon",
    href: "https://telegram.org",
    target: "_blank",
  },
  {
    icon: FacebookIcon,
    className: "social-icon",
    href: "https://www.facebook.com",
    target: "_blank",
  },
  {
    icon: EmailIcon,
    className: "social-icon",
    href: "mailto:someone@example.com",
    target: "_blank",
  },
  {
    icon: PhoneIcon,
    className: "social-icon",
    href: "tel:+123456789",
    target: "_blank",
  },
];

export default function ContactInfo() {
  return (
    <div className="contact-section">
      <h3>Contact Information</h3>
      <nav className="contact-nav">
        {contactInfo.map((item, index) => (
          <a
            key={index}
            href={item.href}
            target={item.target}
            rel="noopener noreferrer"
          >
            <img src={item.icon} alt="" className={item.className} />
          </a>
        ))}
        <a
          href="https://teamchallenge.io/team/387/public"
          className="footer-link"
          target="_blank"
          rel="noopener noreferrer"
        >
          © 2024 GadgetHouse
        </a>
      </nav>
    </div>
  );
}

Reply:
<img width="534" alt="Screenshot 2024-07-07 at 18 36 12" src="https://github.com/Artur0412-KU/online-shop/assets/106263714/be706091-4df7-4bb8-bc29-d2269b6e5ac6">
<img width="561" alt="Screenshot 2024-07-07 at 18 36 48" src="https://github.com/Artur0412-KU/online-shop/assets/106263714/dab7a9ac-8b14-4c28-854c-78e838e52463">
Something like this?